configs/config-no-entropy.h

```diff
@@ -82,6 +82,7 @@
 #define MBEDTLS_X509_USE_C
 #define MBEDTLS_X509_CRT_PARSE_C
 #define MBEDTLS_X509_CRL_PARSE_C
+//#define MBEDTLS_CMAC_C
```

Comment:
Having a commented out #define looks a little odd on its own. Could you consider adding a comment?